### PR TITLE
fix(telegraf): describe gpg key compatibility (closes #5231)

### LINKED ISSUE #5231: APT key for Telegraf varies with Ubuntu version.
APT key for Telegraf varies with Ubuntu version.

Documentation doesn’t distinguish between Ubuntu version.

##### Relevant URLs
- https://docs.influxdata.com/telegraf/v1/install/

### PR DESCRIPTION
**Pull request by jstirnaman**
Closes #5231

- Add example and description of newer vs older GPG key.
- Update test.Dockerfile to use the newer key.

### CODE REVIEW

File: content/telegraf/v1/install.md [hunk 1]
```diff
@@ -22,6 +22,23 @@ To install Telegraf, do the following:
...
+
+InfluxData uses [GPG (GnuPG)](https://www.gnupg.org/software/) to sign released software and provides
+public key and encrypted private key (`.key` file) pairs that you can use to verify the integrity of downloads from the InfluxData repository.
+
+Before running the [install](#install) sample code, substitute the key-pair compatible with your OS version:
+
+For newer OS releases (for example, Ubuntu 20.04 LTS and newer) that support subkey verification:
+
+-  Private key file: `influxdata-archive.key`
+-  Public key: 943666881a1b8d9b849b74caebf02d3465d6beb716510d86a39f6c8e8dac7515
```
Comment (sanderson) [inline content/telegraf/v1/install.md:35]:
```suggestion
-  Public key: `943666881a1b8d9b849b74caebf02d3465d6beb716510d86a39f6c8e8dac7515`
```

File: content/telegraf/v1/install.md [hunk 2]
```diff
@@ -22,6 +22,23 @@ To install Telegraf, do the following:
...
+
+For all Ubuntu and Debian versions, including older versions (for example, CentOS/RHEL 7, Ubuntu 18.04 LTS, or Debian Buster) that don't support subkeys for verification:
+
+-  Private key file: `influxdata-archive_compat.key`
+-  Public key: 393e8779c89ac8d958f81f942f9ad7fb82a25e133faddaf92e15b16e6ac9ce4c
```
Comment (sanderson) [inline content/telegraf/v1/install.md:40]:
```suggestion
-  Public key: `393e8779c89ac8d958f81f942f9ad7fb82a25e133faddaf92e15b16e6ac9ce4c`
```

**Review by sanderson — APPROVED** [review]
Looks good. Minor suggestions.

**Review by powersj — APPROVED** [review]
Thank you!

**Pull request merged**